trin-core/src/portalnet/overlay.rs

```diff
@@ -398,30 +403,18 @@ impl<TContentKey: OverlayContentKey + Send, TMetric: Metric + Send>
             message: content_items,
         };
 
-        let utp_listener = self.utp_listener.clone();
         tokio::spawn(async move {
-            if let Some(conn) = utp_listener
-                .write()
+            // send the content to the acceptor over a uTP stream
+            if let Err(msg) = conn
```

Comment:
nit: I think `err` is a more clear variable name here and in other `Err` matches.